### DISCUSSION TITLE
Guidance for hiring a small proof-of-concept feature development

### DISCUSSION THREAD

**Discussion by 123AndGo** [AnySoftKeyboard/AnySoftKeyboard#2945] [General]
Hello everybody, i was wondering if somebody would be interested to be hired to develop me a small proof-of-concept feature related to translations and text-to-speech.

If nobody would be interested, i would also appreciate if you could advise me on the best and fastest way where i could find a developer that could help me (e.g. a freelancers' site or a development company, etc.).

I would really appreciate it!

**Comment by menny** [comment]
I can help you with guidance. What do you need? You can also email me (my address is on my profile)

**Reply by 123AndGo** [reply]
Thank you very much Menny! I just wrote you to your email with the details.

But in general i am looking if somebody could recommend me some freelancer or company that i could hire to develop me a proof-of-concept feature on top of the AnySoftKeyboard, maybe someone who has already developed something for the AnySoftKeyboard.

Again, thank you a lot!
